Updated Training Page (#90)
ELIXIR-NO/website: src/content/training/index.mdx

<link rel="stylesheet" property="stylesheet" href="https://elixirtess.github.io/TeSS_widgets/css/tess-widget.css"/>

# Training by ELIXIR Norway
ELIXIR Norway aims to provide high-quality bioinformatics training and a wide set of services, designed to support researchers and professionals in life science research, to both the local and global community. Our training programs provide comprehensive resources to enhance your skills and knowledge, enabling you to stay at the forefront of cutting-edge scientific advancements. Regardless of the experience you have as scientist or data analyst, we offer a diverse array of courses, workshops and materials tailored to meet your specific needs, all crafted by instructors with relevant expertise and competency. Our trainings materials are open-access. 

## Types of Training
- **Research Data management (RDM) courses**  
We offer general RDM courses covering a range of topic from data collection and metadata management to understanding regulations for handling sensitive data. In addition, we deliver courses on what best practices to consider for FAIR data handling.

- **Tool-specific courses**  

> We offer courses designed to provide hands-on experience on tools developed (by ELIXIR Norway) for your data management tasks such as [Norwegian e-Infrastructure for Life Sciences](https://nels.bioinfo.no/) for storage of non-sensitive data, [UseGalaxy](https://galaxyproject.org/use/usegalaxy-no/) where you can use/create streamlined bioinformatics workflows, [local Data Stewardship Wizard instance](https://norway.dsw.elixir-europe.org/wizard/) that you can use to create Data Management Plans based on domain-specific templates relevant to the field of your research project, and [FEGA-Norway](https://ega.elixir.no/) that allows secure sharing of sensitive data across borders in compliance with data governance procedures including GDPR regulations in Norway. 

Reviewer: ```suggestion
We offer courses designed to provide hands-on experience on tools for your data management tasks such as [Norwegian e-Infrastructure for Life Sciences](https://nels.bioinfo.no/) for storage of non-sensitive data, [UseGalaxy](https://galaxyproject.org/use/usegalaxy-no/) where you can use/create streamlined bioinformatics workflows, [national Data Stewardship Wizard instance](https://norway.dsw.elixir-europe.org/wizard/) that you can use to create Data Management Plans based on domain-specific templates relevant to the field of your research project, and [FEGA-Norway](https://ega.elixir.no/) that allows secure sharing of sensitive data across borders in compliance with data governance procedures including GDPR in Norway. 
```

These courses are designed to help you navigate and make the most of these tools, improving your data management practices and supporting your research objectives. Whether you are looking to get started or deepen your expertise, our training courses are structured to meet the diverse needs of researchers at all levels.

- **Domain-specific courses**  
In addition to tool-specific training, ELIXIR Norway offers a range of domain-specific courses aimed at enhancing your expertise in specialised areas of bioinformatics and data analysis. These courses cover key topics such as single-cell sequencing, microbiome analysis, biodiversity studies, and sequencing assemblies. Our courses focus on challenges and techniques to help explore and analyse data of your interest and apply best practices for assembling high-quality sequences, from short-read to long-read technologies. These domain-specific courses are designed to provide you with the knowledge and skills to tackle the specific challenges in your research projects with confidence.

- **Train-the-Trainer**  

> As a national ELIXIR node, we are a part of ELIXIR-GOBLET Train-the-Trainer (TtT) programme that enables us to provide training on how to become a better instructor/trainer. The TtT courses touch upon various teaching techniques that can help you to ensure productive engagement from learners/trainees and make your course material easy to understand. TtT is an open instructors’ community and you can read more about it here https://elixir-europe.org/platforms/training/train-the-trainer

Reviewer: ```suggestion
As a national ELIXIR node, we are a part of the ELIXIR-GOBLET Train-the-Trainer (TtT) programme that enables us to provide training on how to become a better instructor/trainer. The TtT courses touch upon various teaching techniques that can help you ensure productive engagement from learners/trainees and make your course material easy to understand. TtT is an open instructors’ community, and you can read more about it [here](https://elixir-europe.org/platforms/training/train-the-trainer)
```

## Get Started

> Navigating ELIXIR's Training programs by exploring our catalogue of available courses, which are categorised by topic and proficiency level to help you find the right fit. Each course page provides detailed information about the curriculum, prerequisites, and learning outcomes. We announce our courses and upload training materials to also [ELIXIR TeSS](https://tess.elixir-europe.org/) where you can also find upcoming trainings that are happening internationally. In addition, you can look at the past training material on our GitHub and Zenodo repositories.

Reviewer: Add links to GH and Zenodo

- **Train-the-Trainer**  
As a national ELIXIR node, we are a part of the ELIXIR-GOBLET Train-the-Trainer (TtT) programme that enables us to provide training on how to become a better instructor/trainer. The TtT courses touch upon various teaching techniques that can help you ensure productive engagement from learners/trainees and make your course material easy to understand. TtT is an open instructors’ community, and you can read more about it [here](https://elixir-europe.org/platforms/training/train-the-trainer)

## Get Started

> Navigating ELIXIR's Training programs by exploring our catalogue of available courses, which are categorised by topic and proficiency level to help you find the right fit. Each course page provides detailed information about the curriculum, prerequisites, and learning outcomes. We announce our courses and upload training materials to also [ELIXIR TeSS](https://tess.elixir-europe.org/) where you can also find upcoming trainings that are happening internationally. In addition, you can look at the past training material on our [GitHub](https://github.com/ELIXIR-Norway-Training) and [Zenodo](https://zenodo.org/communities/elixir-no) repositories.

Reviewer: ```suggestion
Navigating ELIXIR's Training programs by exploring our catalogue of available courses, which are categorised by topic and proficiency level to help you find the right fit. Each course page provides detailed information about the curriculum, prerequisites, and learning outcomes. We announce our courses and upload training materials to also [ELIXIR TeSS](https://tess.elixir-europe.org/) where you can also find upcoming trainings that are happening internationally. In addition, you can look at the past training material on our [GitHub](https://github.com/ELIXIR-Norway-Training) and [Zenodo](https://zenodo.org/communities/elixir-no/records?q=&f=resource_type%3Alesson&f=resource_type%3Apresentation&l=list&p=1&s=10&sort=newest) repositories.
```
